src/Symfony/Component/Validator/Constraints/Range.php

```diff
@@ -60,6 +71,15 @@ public function __construct($options = null)
             if ((isset($options['minPropertyPath']) || isset($options['maxPropertyPath'])) && !class_exists(PropertyAccess::class)) {
                 throw new LogicException(sprintf('The "%s" constraint requires the Symfony PropertyAccess component to use the "minPropertyPath" or "maxPropertyPath" option.', static::class));
             }
+
+            if (isset($options['min']) && isset($options['max'])) {
+                $this->deprecatedMinMessageSet = isset($options['minMessage']);
+                $this->deprecatedMaxMessageSet = isset($options['maxMessage']);
+
+                if ($this->deprecatedMinMessageSet || $this->deprecatedMaxMessageSet) {
+                    @trigger_error('Since symfony/validator 4.4: "minMessage" and "maxMessage" are deprecated when the "min" and "max" options are both set. Use "notInRangeMessage" instead.', E_USER_DEPRECATED);
```

Comment:
It means we also miss an exception here in 5.0, and maybe that should be part of the message.

Reply:
I'm not sure to understand, what should be part of the message please ?

Reply:
```Since symfony/validator 4.4: "minMessage" and "maxMessage" are deprecated when the "min" and "max" options are both set. You should use "notInRangeMessage" instead or an exception will be thrown in 5.0.`

Reply:
But we cannot add an Exception here in a patch release of 5.0.x, right? This would be quite a BC break

Reply:
well, we have a default `notInRangeMessage` in 5.0. So 5.0 will not throw an exception. It will use the built-in message even you you might have intended to use a custom one.

Reply:
>even if you might have intended to use a custom one

Sounds like a silent failure, the deprecation should be replaced by an exception in 5.0 or master IMO because the intended message won't be used.